Jenkinsfile

```diff
@@ -805,12 +803,18 @@ def doBuildApplePlatform(String platform, String buildType, boolean test = false
                         if (platform != 'iphonesimulator') error 'Testing is only available for iOS Simulator'
                         sh "xcodebuild -scheme CoreTests -configuration ${buildType} -destination \"${buildDestination}\""
                         def env = environment().collect { v -> "SIMCTL_CHILD_${v}" }
+                        env << "SIMCTL_CHILD_UNITTEST_XML=${WORKSPACE}/build-dir/test/unit-test-report.xml"
```

Comment:
How does this work, are all the env variables with the SIMCTL_CHILD_ prefix forwarded through to the simulator environment?

Reply:
Yeah, the simulator doesn't inherit env variables by default but anything with the `SIMCTL_CHILD_` prefix is set in the simulator process with that prefix stripped.